scripts/msb-tests.groovy

```diff
@@ -0,0 +1,63 @@
+node("cirhos_rhel7"){
+      deleteDir()
+
+      withEnv(["GOPATH=$WORKSPACE",
+               "BROKER_URL=http://msb-${NAMESPACE}.apps.${CLUSTER_ROUTE}"
+            ])
+            {
+                  stage('Clone the managed-service-broker repo'){
+                        dir(".") {
+                              git branch: 'master', url: 'https://github.com/integr8ly/managed-service-broker.git'
+                              if(!fileExists(file: './Makefile')) error("Error cloning managed-service-broker repo")
+                        } // dir
+                  }
+        
+                  stage('Docker Build & Push'){
+                        dir("$GOPATH/src/github.com/integr8ly"){
+                              sh '''
+                                    ln -sf $WORKSPACE managed-service-broker
+                                    cd managed-service-broker
+            
+                                    make build_binary 
+                                    sudo docker login -u ${QUAY_USERNAME} -p ${QUAY_PASSWORD} quay.io
+                                    sudo docker build -t quay.io/integr8lyqe/managed-services-broker:1.0.0 -f ./tmp/build/broker/Dockerfile .
+                                    sudo docker push quay.io/integr8lyqe/managed-services-broker:1.0.0
+                              '''
```

Comment:
Would it be possible to use different tag? `1.0.0` seems as if it is stable version but it is just what is currently in master branch

Reply:
which tag do you suggest? latest?:)

Reply:
Yes, `latest` sound good. Other options are `master` or even something like `0.0.1`. I leave it up to you, we can stick with `latest`, that's fine.